lms/templates/staff_problem_info.html

```diff
@@ -77,6 +77,8 @@
         % if can_rescore_problem:
         |
         <button type="button" class="btn-link staff-debug-rescore">${_('Rescore Student Submission')}</button>
+        |
```

Comment:
Yes.  They are on the Staff Debug Info view of the problem.
<img width="585" alt="screen shot 2016-10-19 at 10 20 34 pm" src="https://cloud.githubusercontent.com/assets/6772826/19544092/5dad0d56-964a-11e6-946e-8a19f21942d0.png">

Rename to the same as the other buttons for Specific Learner?